src/App.tsx

```diff
@@ -12,11 +20,15 @@ function App() {
     roles: ["director_producer"],
   };
   return (
-    <div className="App">
-      <header className="App-header">
-        <VideoCard {...video} />
```

Comment:
can we render this `VideoCard` component somewhere for the time being? Probably home right now so we can mess with it easily.